lib/relation-definition.js

```diff
@@ -1247,6 +1286,8 @@ RelationDefinition.belongsTo = function(modelFrom, modelTo, params) {
 
     modelFrom.dataSource.defineProperty(modelFrom.modelName, discriminator, {type: 'string', index: true});
   } else {
+    // relation is not polymorphic
+    modelTo = lookupModelTo(modelFrom, modelToRef, params);
```

Comment:
given the refactor there's no need really. `lookUpModelTo` will do its best to find the model from `modelToRef` and `params`. 

if `modelToRef` is already a model it will passthrough and return it right away.

The check was required when `modelTo` was used instead of `modelToRef`.